transcendence-app/src/user/user.domain.ts

```diff
@@ -1,9 +1,21 @@
 export class User {
+  username: string;
+  email: string;
+  avatar_id: string | null;
+  id: string;
+  created_at: Date;
+
   constructor(
-    public username: string,
-    public email: string,
-    public avatar: string | null,
-    public id: string,
-    public created_at: number,
-  ) {}
```

Comment:
The [Swagger CLI](https://docs.nestjs.com/openapi/cli-plugin) plugin doesn't work with this class syntax. I've changed to a more verbose format to make it work and automatically detect the scheme of the class.